cmd/kubectl/kubectlcobra/basic_printer.go

```diff
@@ -0,0 +1,47 @@
+package kubectlcobra
+
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"sigs.k8s.io/kustomize/kstatus/wait"
+)
+
+type BasicPrinter struct {
+	ioStreams genericclioptions.IOStreams
+}
+
+func (b *BasicPrinter) Print(ch <-chan Event) {
+	for event := range ch {
+		switch event.EventType {
+		case ErrorEventType:
+			cmdutil.CheckErr(event.ErrorEvent.Err)
+			return
```

Comment:
why return here but not elsewhere?

Reply:
It is not needed. I have updated the PR.